index.js

```diff
@@ -134,6 +134,9 @@ function toRunCommand (inspectObj, name) {
   if (cfg.ExposedPorts && isCompatible('--expose', modes)) {
     rc = appendObjectKeys(rc, '--expose', cfg.ExposedPorts)
   }
+  if (cfg.Labels && isCompatible('--label', modes)) {
+    rc = appendObjectEntries(rc, '--label', cfg.Labels, '=')
```

Comment:
Can we just use `-l` instead of `--label`?

Reply:
Done